examples/Remarketing/UploadCallConversion.cs

```diff
@@ -0,0 +1,121 @@
+        /// <summary>
+        /// Main method, to run this code example as a standalone application.
+        /// </summary>
+        /// <param name="args">The command line arguments.</param>
+        public static void Main(string[] args)
+        {
+            UploadCallConversion codeExample = new UploadCallConversion();
+            Console.WriteLine(codeExample.Description);
+
+            // The Google Ads customer ID for which the call is made.
+            long customerId = long.Parse("INSERT_CUSTOMER_ID_HERE");
+
+            // The caller ID.
+            string callerId = "INSERT_CALLER_ID_HERE";
+
+            // The call start time ID.
+            string callStartTime = "INSERT_CALL_START_TIME_HERE";
+
+            // The conversion time.
+            string conversionTime = "INSERT_CONVERSION_TIME_HERE";
+
+            // The conversion value.
+            double conversionValue = double.Parse("CONVERSION_VALUE");
+
+            codeExample.Run(new GoogleAdsClient(), customerId, callerId, callStartTime,
+                conversionTime, conversionValue);
+        }
+
+        /// <summary>
+        /// Returns a description about the code example.
+        /// </summary>
+        public override string Description
+        {
+            get
+            {
+                return "This code example imports offline call conversion values for calls " +
+                    "related to the ads in your account. To set up a conversion action, run the " +
+                    "AddConversionAction.cs example.";
+            }
+        }
+
+        /// <summary>
+        /// Runs the code example.
+        /// </summary>
+        /// <param name="client">The Google Ads client.</param>
+        /// <param name="customerId">The Google Ads customer ID for the conversion action is
+        /// added.</param>
+        /// <param name="callerId">The caller ID.</param>
+        /// <param name="conversionTime">The call conversion time.</param>
+        /// <param name="callStartTime">The call start time.</param>
+        /// <param name="conversionValue">The convsersion value.</param>
+        public void Run(GoogleAdsClient client, long customerId, string callerId,
+            string callStartTime, string conversionTime, double conversionValue)
+        {
+            // Get the ConversionActionService.
+            ConversionUploadServiceClient conversionUploadService =
+                client.GetService(Services.V3.ConversionUploadService);
+
+            // Create a call conversion by specifying currency as USD.
+            CallConversion callConversion = new CallConversion()
+            {
+                CallerId = callerId,
+                CallStartDateTime = callStartTime,
+                ConversionDateTime = conversionTime,
+                ConversionValue = conversionValue,
+                CurrencyCode = "USD"
+            };
+
+            try
+            {
+                // Issues a request to upload the call conversion.
```

Comment:
Not sure if these are mandatory specifically here but I think it is worth adding a comment to explain the "true" and "false" parameter values.

Reply:
Done.